### PR TITLE
SSCS-9588-ImplementKustomize_SSCS_EvidenceShare

### PR DESCRIPTION
**Pull request by rmurphy-hmcts**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/SSCS-9588

### Change description ###
SSCS-evidence share migrated to FluxV1
Migration of application CCD evidence share in SSCS Namespaces to Kustomize.
Application currently running in aat, demo, ithc, and perftest clusters.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: k8s/namespaces/sscs/sscs-evidence-share/demo.yaml [hunk 1]
```diff
@@ -5,18 +5,11 @@ metadata:
...
-  chart:
-    git: git@github.com:hmcts/hmcts-charts
-    ref: master
-    path: stable/sscs-evidence-share
   values:
     java:
       replicas: 2
       useInterpodAntiAffinity: true
-      image: hmctspublic.azurecr.io/sscs/evidence-share:pr-1301-226854b-20220510105503 #{"$imagepolicy": "flux-system:demo-sscs-evidence-share"}
+      image: hmctspublic.azurecr.io/sscs/evidence-share:pr-1283-9665907-20220504171554 #{"$imagepolicy": "flux-system:demo-sscs-evidence-share"}
```
Comment (dattipoeHMCTS) [inline k8s/namespaces/sscs/sscs-evidence-share/demo.yaml:12]: Please don't override the tag changes

**Review by dattipoeHMCTS — CHANGES_REQUESTED** [review]

File: k8s/namespaces/sscs/sscs-evidence-share/sscs-evidence-share.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  namespace: sscs
+spec:
+  releaseName: sscs-evidence-share
+  rollback:
+    enable: true
+    retry: true
+  chart:
+    git: git@github.com:hmcts/hmcts-charts
+    ref: master
+    path: stable/sscs-evidence-share
```
Comment (dattipoeHMCTS) [inline k8s/namespaces/sscs/sscs-evidence-share/sscs-evidence-share.yaml:14]: A lot more could be brought into the base chart

**Review by dattipoeHMCTS — CHANGES_REQUESTED** [review]

File: k8s/namespaces/sscs/sscs-evidence-share/demo.yaml (hunk 1 above)
Comment (rmurphy-hmcts) [reply k8s/namespaces/sscs/sscs-evidence-share/demo.yaml]: Image tag updated

File: k8s/namespaces/sscs/sscs-evidence-share/sscs-evidence-share.yaml (hunk 2 above)
Comment (rmurphy-hmcts) [reply k8s/namespaces/sscs/sscs-evidence-share/sscs-evidence-share.yaml:14]: Further sections added to base chart

**Pull request closed without merging**